Main.py

```diff
@@ -1,8 +1,9 @@
 from preprocessing.DataCleanser import runDataCleanser
-from preprocessing.DataLoader import runDataLoader
+from preprocessing.DataLoader import runDataLoader, runQuickLoader
 from preprocessing.WordCounter import addAbstractAndBodyWordCountColumn
 
 covidDF = runDataLoader()
+# covidDF = runQuickLoader()
```

Comment:
@redhed976 If you want to quickly go through the data, just uncomment this and comment out the line above this.